### PR TITLE
Adding reef to upstream listener job

### PR DESCRIPTION
**Pull request by sunilkumarn417**
Adding reef branch into upstream listener build to collect reef build infromation.

https://jenkins.ceph.redhat.com/job/rhceph-upstream-listener/869/console

### CODE REVIEW

File: pipeline/upstream_listener.groovy [hunk 1]
```diff
@@ -84,6 +84,11 @@ node("rhel-9-medium") {
                 job: "rhceph-upstream-listener",
                 parameters: [string(name: 'releaseName', value: "quincy")]
             ])
```
Comment (anrao19) [inline pipeline/upstream_listener.groovy:86]: @sunilkumarn417, IMO we need to include else if condition before build
else if ( upstreamVersion == "quincy" )

Comment (sunilkumarn417) [reply pipeline/upstream_listener.groovy:86]: I think we can start two jobs from main one.

Comment (sunilkumarn417) [reply pipeline/upstream_listener.groovy:86]: its working, https://jenkins.ceph.redhat.com/job/rhceph-upstream-listener/869/console

File: pipeline/upstream_listener.groovy [hunk 2]
```diff
@@ -32,7 +32,7 @@ node("rhel-9-medium") {
                             reference: ''
                         ]],
                         userRemoteConfigs: [[
-                            url: 'https://github.com/red-hat-storage/cephci.git'
+                            url: 'https://github.com/sunilkumarn417/cephci.git'
```
Comment (anrao19) [inline pipeline/upstream_listener.groovy:35]: Please update the url, other than that changes looks good to me

**Review by vamahaja — APPROVED** [review]

**Pull request merged**